src/libs/Navigation/types.ts

```diff
@@ -372,8 +372,8 @@ type PublicScreensParamList = {
         validateCode: string;
     };
     [SCREENS.UNLINK_LOGIN]: {
-        accountID: string;
-        validateCode: string;
+        accountID?: string;
+        validateCode?: string;
```

Comment:
Since `UnlinkLoginPage` had default values for these two params, I suggest they are optional. Otherwise, I can make it required but then it makes sense to get rid of default values here
```
const accountID = route.params.accountID ?? '';
const validateCode = route.params.validateCode ?? '';
```
 which I'm not sure will not lead to regression...